backend/app/views/spree/admin/products/index.html.erb

```diff
@@ -34,7 +34,7 @@
           <div class="col-2">
             <div class="field checkbox">
               <label>
-                <%= f.check_box :deleted_at_null, {checked: params[:q][:deleted_at_null] == '0'}, '0', '1' %>
+                <%= f.check_box :with_deleted, {checked: params[:q][:with_deleted] == 'true'}, 'true', 'false' %>
```

Comment:
I'd take advantage of this change to add spaces inside `{}` to be consistent with the rest of the code .